aviary/mission/gasp_based/phases/time_integration_traj.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from aviary.mission.gasp_based.ode.time_integration_base_classes import SGMTrajBase
 from aviary.mission.gasp_based.phases.time_integration_phases import SGMGroundroll, SGMRotation
+from aviary.variable_info.enums import Verbosity
```

Comment:
This is the first example I've seen so far of where you've added Verbosity back instead of using a numerical assignment like what you did in test_linkage_logic.py. Was this on purpose or was this an oversight?